node/nodeBlocks.go

```diff
@@ -35,6 +35,15 @@ func (n *Node) GetBlockByNonce(nonce uint64, withTxs bool) (*api.Block, error) {
 	return apiBlockProcessor.GetBlockByNonce(nonce, withTxs)
 }
 
+func (n *Node) GetBlockByRound(round uint64, withTxs bool) (*api.Block, error) {
+	apiBlockProcessor, err := n.createAPIBlockProcessor()
```

Comment:
@miiu96 please check this. Why do we need to create an apiBlockProcessor for **each request**? We should definitely change this.

Reply:
@miiu96 ?